v2/manifests/core-ons/the-train.yml

```diff
@@ -19,7 +19,6 @@ services:
       PORT:                        8084
       DP_COLOURED_LOGGING:         "true"
       DP_LOGGING_FORMAT:           "pretty_json"
-      PUBLISHING_THREAD_POOL_SIZE: 100
```

Comment:
Why are you removing this?

Reply:
This was in added tow times so one was redundant.